ui/app/src/Main.elm

```diff
@@ -8,7 +8,7 @@ import Types
         ( Route(..)
         , Msg
             ( NavigateToSilenceList
-            , NavigateToSilence
+            , NavigateToSilenceView
```

Comment:
Not a big thing: Why adding `View` to `NavigateToSilence` msg? We don't do it with the other ones.